zerver/lib/actions.py

```diff
@@ -6271,18 +6282,18 @@ def do_get_user_invites(user_profile: UserProfile) -> List[Dict[str, Any]]:
         # We do not return multiuse invites to non-admin users.
         return invites
 
-    lowest_datetime = timezone_now() - datetime.timedelta(
-        days=settings.INVITATION_LINK_VALIDITY_DAYS
-    )
+    lowest_datetime = timezone_now() - datetime.timedelta(days=1) * F("validity_in_days")
```

Comment:
If we used the `expiration_date` model that Tim suggested, we  would a straight-forward query, here, right? I'm inclined to agree that we should go with that, for the sake of simplicity.

Reply:
Yeah, queries will be certainly straight-forward and clean.